README.md

```diff
@@ -24,5 +24,169 @@ Note that the RTI website has [Free Trial](https://www.rti.com/free-trial) offer
 ## Building
 Refer to the [Install DDS Implementations](https://index.ros.org/doc/ros2/Installation/DDS-Implementations) page for details on building rmw_connext for your platform.  
 
-### Quality Declaration (per [REP-2004](https://github.com/ros-infrastructure/rep/blob/master/rep-2004.rst))
+## Using Connext XML QoS settings
+
+QoS profiles can be specified in XML according to the load order specified [here](https://community.rti.com/static/documentation/connext-dds/5.2.0/doc/manuals/connext_dds/html_files/RTI_ConnextDDS_CoreLibraries_UsersManual/Content/UsersManual/How_to_Load_XML_Specified_QoS_Settings.htm). `url_profile` and `string_profile` cannot be used.
+
+ROS will use the profile with the `is_default_qos="true"` attribute.
+That profile will be used as a base and it will be overridden with the ROS specific policies except when `rmw_qos_profile_system_default` is used.
+For example:
+
+```xml
+<?xml version="1.0"?>
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/5.3.1/rti_dds_qos_profiles.xsd" version="5.3.1">
+  <qos_library name="Ros2TestQosLibrary">
+    <qos_profile name="Ros2TestDefaultQos" base_name="BuiltinQosLib::Baseline.5.3.0" is_default_qos="true">
+      <participant_qos>
+        <property>
+          <value>
+            <!-- 6.25 MB/sec (52 Mb/sec) flow controller -->
+            <element>
+              <name>dds.flow_controller.token_bucket.slow_flow.token_bucket.max_tokens</name>
+              <value>8</value>
+            </element>
+            <element>
+              <name>dds.flow_controller.token_bucket.slow_flow.token_bucket.tokens_added_per_period</name>
+              <value>8</value>
+            </element>
+            <element>
+              <name>dds.flow_controller.token_bucket.slow_flow.token_bucket.bytes_per_token</name>
+              <value>8192</value>
+            </element>
+            <element>
+              <name>dds.flow_controller.token_bucket.slow_flow.token_bucket.period.sec</name>
+              <value>0</value>
+            </element>
+            <element>
+              <name>dds.flow_controller.token_bucket.slow_flow.token_bucket.period.nanosec</name>
+              <value>10000000</value>
+            </element>
+          </value>
+        </property>
+      </participant_qos>
+
+      <datawriter_qos topic_filter="rt/my_large_data_topic">
+        <publish_mode>
+          <flow_controller_name>dds.flow_controller.token_bucket.slow_flow</flow_controller_name>
+        </publish_mode>
+      </datawriter_qos>
+    </qos_profile>
+  </qos_library>
+</dds>
+```
+
+That will force all publishers in the `my_large_data_topic` to use the `slow_flow` flow controller.
+See `Topic Name Mangling` section to understand the `rt/` prefix.
```

Comment:
where do I find the `Topic Name Mangling` section?

Reply:
Sorry, I forgot writing it.

Reply:
Added in https://github.com/ros2/rmw_connext/pull/460/commits/db3b1d08367ba9f580977850e0631300d8a05630.